MIGRATIONS.md

```diff
@@ -29,3 +29,24 @@ mysql -ss -u root -p ttm-enlace -e "SELECT Program_Date_ID, Program_ID FROM Prog
 done
 
 ```
+
+## Issue 218 - Enlace - Hiding past data - September 2018
+
+Adds a settings table for the enlace subsystem.  This is a generic placeholder for any settings that can be updated by administrators that affect how the application works.  The relevant page is system/settings.php
+
+### SQL for Table Creation
+```
+USE ttm-enlace
+CREATE TABLE `System_Settings` (
+    `Setting_Name` varchar(255) NOT NULL UNIQUE,
+    `Setting_Type` enum('integer', 'string') NOT NULL,
+    `Setting_Value` varchar(255)
+);
+```
+
+### num_days_hidden setting
+System setting for number of days, for which if a program's sessions all end after this, will be hidden.
+```
+USE ttm-enlace
+INSERT INTO System_Settings VALUES ('num_days_hidden', 'integer', 365);
```

Comment:
This is probably just my ignorance about what the `Setting_Type` enum means relative to the `Setting_Value` column, but: how is it that we're putting a raw integer into a varchar(255) field?

Reply:
Ooo, mysql has implicit type conversion, woo!  Setting_Type is used by the php code for casting after it gets the data out of mysql.